### PR TITLE
Update Multiple inheritance in basics.rst

### PR DESCRIPTION
**Pull request by bcarmo-caio**
```
$ python --version
Python 3.9.18

$ pip freeze | grep colander
colander==2.0
```

### CODE REVIEW

**Comment by bcarmo-caio** [comment]
Or perhaps

```
class Three(Two, One):
```

was meant. And then the output matches.

**Comment by tseaver** [comment]
After I get the CI unstuck, I'll look more closely at the output here.

File: docs/basics.rst [hunk 1]
```diff
@@ -773,11 +773,11 @@ Here's what's printed when the above is run:
...
-     <colander.Boolean object at 0xe2e110>),
-    (<colander.SchemaNode object at 14868688 (named c)>,
-     <colander.String object at 0xe2e090>),
+     <colander.Integer object at 0xe25f90>),
+    (<colander.SchemaNode object at 14868816 (named c)>,
+     <colander.String object at 0xe2e110>),
+    (<colander.SchemaNode object at 14868688 (named b)>,
+     <colander.Boolean object at 0xe2e090>),
     (<colander.SchemaNode object at 14868944 (named d)>,
      <colander.Boolean object at 0xe2e190>)]
```
Comment (tseaver) [inline docs/basics.rst:783]: I think we need to add an explanatory comment that the order of the children in the output is governed by Python's `__mro__`, e.g.
```suggestion

.. note::

   The order of the children in the output above might seem suprising:
   it is governed by how Python's multiple-inheritance works (the
   ``__mro__`` order).
```

**Review by tseaver — CHANGES_REQUESTED** [review]
Thanks very much for the patch!  I've fixed the CI issue already, so this one should be ready to merge once the note I suggested is added.

**Pull request merged**